templates/events/add-new-event.html

```diff
@@ -574,6 +574,24 @@ <h5>Display Only with Super Event</h5>
     <script src="https://cdn.quilljs.com/1.3.6/quill.js"></script>
 
    <script>
+   function changeImage(element, img) {
```

Comment:
Since its implemented this way, the :focus and :active styles are missing. 
The contributions/catalog also has the same feature. It is implemented with css stylings. Maybe can take a look.
https://github.com/rokwire/rokwire-building-blocks-api/blob/develop/contributions/catalog/webapps/static/css/style.css#L39
https://github.com/rokwire/rokwire-building-blocks-api/blob/develop/contributions/catalog/webapps/static/css/style.css#L48
https://github.com/rokwire/rokwire-building-blocks-api/blob/develop/contributions/catalog/webapps/templates/topnav.html#L11

Reply:
This should have been resolved by https://github.com/rokwire/events-manager/pull/1074/commits/f5cb4f36c9e6d6065507492256073dd0d8dc6dcd